packages/api-catalog-service/tslint.json

```diff
@@ -0,0 +1,113 @@
+{
```

Comment:
can't we migrate to eslint with ts rather than [tslint](https://www.npmjs.com/package/tslint) as its depreciated.

Reply:
updated